src/libponyrt/sched/start.c

```diff
@@ -141,7 +141,13 @@ PONY_API int pony_start(bool library, bool language_features)
   if(library)
     return 0;
 
-  return atomic_load_explicit(&exit_code, memory_order_relaxed);
+  if(language_init)
+  {
+    ponyint_os_sockets_final();
+    language_init = false;
+  }
```

Comment:
This looks like it was an additional fix not described in the ticket description?